lib/modules/solidity/solcW.js

```diff
@@ -39,30 +39,32 @@ class SolcW {
       logger: self.logger,
       events: self.events
     });
-    this.solcProcess.send({action: "init", options: {}});
 
-    this.solcProcess.once('result', 'loadedCompiler', () => {
+    this.solcProcess.once("result", "initiated", () => {
+      this.events.request("version:get:solc", function(solcVersion) {
+        if (solcVersion === currentSolcVersion) {
+          self.solcProcess.send({action: 'loadCompiler', requirePath: 'solc'});
```

Comment:
Just `return` here and then you can get rid of the `else`

Reply:
Will fix, but functionally this is the same, no? What do you have against `else`'s 🤣

Reply:
Fixed

Reply:
I HATE elses 😠 
Haha jk. Just that they had a level of depth in the code that is useless. Makes the code a bit harder to read.